### PR TITLE
feat: a32nx github link command

### PR DESCRIPTION
**Pull request by owen2007**
## Description

Adds a command which has a link to the a32nx github page

## Test Results

image

## Discord Username

owen_ #4883

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -221,7 +222,8 @@ const commands: CommandDefinition[] = [
     abbreviations,
     airac,
     mico,
-    translate
+    translate,
+    github
```
Comment (oim1) [inline src/commands/index.ts:226]:
```suggestion
    github,
```

File: src/commands/support/github.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    name: 'github',
+    description: 'provides the link to the a32nx github page',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const githubEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | github page',
+            description: makeLines ([
+                'Below is the link to our a32nx github page',
+                '',
+                '[a32nx github page](https://github.com/flybywiresim/a32nx)',
```
Comment (oim1) [inline src/commands/support/github.ts:15]:
```suggestion
                '[A32NX GitHub Repository](https://github.com/flybywiresim/a32nx)',
```

File: src/commands/support/github.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const github: CommandDefinition = {
+    name: 'github',
+    description: 'provides the link to the a32nx github page',
```
Comment (oim1) [inline src/commands/support/github.ts:7]:
```suggestion
    description: 'provides the link to the A32NX GitHub Repository',
```

File: src/commands/support/github.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const githubEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | github page',
+            description: makeLines ([
+                'Below is the link to our a32nx github page',
```
Comment (oim1) [inline src/commands/support/github.ts:13]:
```suggestion
                'Below is the link to our A32NX GitHub Repository',
```

File: src/commands/support/github.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const github: CommandDefinition = {
+    name: 'github',
+    description: 'provides the link to the a32nx github page',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const githubEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | github page',
```
Comment (oim1) [inline src/commands/support/github.ts:11]:
```suggestion
            title: 'FlyByWire A32NX | GitHub Repository',
```

File: src/commands/support/github.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const github: CommandDefinition = {
+    name: 'github',
```
Comment (oim1) [inline src/commands/support/github.ts:6]:
```suggestion
    name: ['github', 'repo', 'repository'],
```

**Review by oim1 — CHANGES_REQUESTED** [review]
Changed a few wording things

File: src/commands/support/github.ts (hunk 6 above)
Comment (owen2007) [reply src/commands/support/github.ts:6]: Will fix all change requests tomorrow! 👍

File: src/commands/support/github.ts [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+export const github: CommandDefinition = {
+    name: ['github', 'repo', 'repository'],
+    description: 'provides the link to the A32NX GitHub Repository',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const githubEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | GitHub Repository',
+            description: makeLines ([
```
Comment (NathanInnes) [inline src/commands/support/github.ts:12]:
```suggestion
            description: makeLines([
```
Remove space.

**Review by NathanInnes — CHANGES_REQUESTED** [review]

File: src/commands/support/github.ts [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+                'Below is the link to our A32NX GitHub Repository',
+                '',
+                '[A32NX GitHub Repository](https://github.com/flybywiresim/a32nx)',
+            ]),
+        });
+
+        await msg.channel.send({ embeds: [githubEmbed] });
+
+    },
+};
```
Comment (NathanInnes) [inline src/commands/support/github.ts:22]:
```suggestion
};

```
Add new line

File: src/commands/support/github.ts [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+            title: 'FlyByWire A32NX | GitHub Repository',
+            description: makeLines ([
+                'Below is the link to our A32NX GitHub Repository',
+                '',
+                '[A32NX GitHub Repository](https://github.com/flybywiresim/a32nx)',
+            ]),
+        });
+
+        await msg.channel.send({ embeds: [githubEmbed] });
+
```
Comment (NathanInnes) [inline src/commands/support/github.ts:20]:
```suggestion
```
Remove padding

Comment (owen2007) [reply src/commands/support/github.ts:20]: Will do tomorrow or today as it 12:01 :/

**Comment by owen2007** [comment]
@benw202 before you merge the PR can you really quickly just make sure that I have completed all of the change requests that Oim and Sabes have requested? Thanks :)

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**